pkg/rbd/nodeserver.go

```diff
@@ -766,10 +772,13 @@ func encryptDevice(ctx context.Context, rbdVol *rbdVolume, cr *util.Credentials,
 		return err
 	}
 
-	imageSpec := rbdVol.Pool + "/" + rbdVol.RbdImageName
-	err = util.SaveRbdImageEncryptionStatus(ctx, cr, rbdVol.Monitors, imageSpec, rbdImageEncrypted)
+	err = rbdVol.ensureEncryptionMetadataSet(rbdImageEncrypted)
+	if err != nil {
+		klog.Errorf(util.Log(ctx, err.Error()))
```

Comment:
use klog.Error as formatting is not required. need to ensure this in other places too

Reply:
Done with the next update